lib/utils.pm

```diff
@@ -2032,4 +2033,19 @@ sub permit_root_ssh {
     }
 }
 
+=head2 permit_root_ssh_in_sol
+    permit_root_ssh_in_sol();
+
+for ipmi backend, PermitRootLogin has to be set in sol console
+however, assert_script_run and script_run is not stable in sole console
+enter_cmd or type_string are acceptable
+
+=cut
+
+sub permit_root_ssh_in_sol {
+
+    my $sshd_config_file = "/etc/ssh/sshd_config";
```

Comment:
Possible to make this part more resilient ? For example,
```
sub permit_root_ssh_in_sol {
       my $sshd_config_file = shift;

       $sshd_config_file //= "/etc/ssh/sshd_config";
}

And also check whether password login is permitted ?

Reply:
> Possible to make this part more resilient ? For example,
> 
> ```
> sub permit_root_ssh_in_sol {
>        my $sshd_config_file = shift;
> 
>        $sshd_config_file //= "/etc/ssh/sshd_config";
> 
>        xxxxx
> }
> ```
Good suggestion.

> 
> And also check whether password login is permitted ?

root can login with password with this change. We'd better not make the function complex as the running commands on sol console is risky.